src/traces/bar/cross_trace_calc.js

```diff
@@ -119,10 +123,44 @@ function setGroupPositions(gd, pa, sa, calcTraces, opts) {
             }
             break;
     }
-
     collectExtents(calcTraces, pa);
 }
 
+function standardizeCornerradius(calcTraces) {
+    if(calcTraces.length < 2) return;
```

Comment:
What happens here if we have one `scatter` and one `bar` traces in data?

Reply:
I believe at this point `calcTraces` contains only bar traces, but I'm not positive -- I will verify

Reply:
Please resolve it if verified?

Reply:
Confirmed: For a plot containing both a scatter trace and a bar trace, `calcTraces` as passed to `standardizeCornerradius()` contains only the bar trace.